decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class Decorator < Nameable
```

Comment:
- [ ] After declaring class , I suggest to use `attr_accessor :nameable` here. Kindly add this.

Reply:
Thanks for the update.
it has been resolved